www/nextcloud-client/Portfile

```diff
@@ -35,18 +32,22 @@ compiler.library_path
 compiler.blacklist-append {clang < 602} macports-clang-3.4 macports-clang-3.3
 compiler.fallback-append macports-clang-5.0
 
```

Comment:
upstream lists the requirement for a C++14 compiler; so perhaps instead of all this blacklisting business you can just request that (i.e., `compiler.cxx_standard 2014`) and see if that is sufficient?

Reply:
I replaced the compiler blacklists with `compiler.cxx_standard 2014`. It works for me on Big Sur, but I don't have other versions to test, unfortunately.